websubhub-ballerina/build.gradle

```diff
@@ -129,17 +153,25 @@ task revertTomlFile {
 
 task ballerinaBuild {
     inputs.dir file(project.projectDir)
+    dependsOn(initializeVariables)
     finalizedBy(revertTomlFile)
 
+    def testParams = "--skip-tests"
+    gradle.taskGraph.whenReady { graph ->
+        if (graph.hasTask(':websubhub-ballerina:test')) {
```

Comment:
Since we don't have disabled the `ballerinaTest` task when the `:build` task is available, this will run the tests twice for each build.
@ThisaruGuruge

Reply:
Just noticed that there is no `ballerinaTest` task at all. Don't we need that to preserve common build process across all StdLibs?

Reply:
Included `ballerinaTest` task to the `:build`.